integrationExamples/gpt/userId_example.html

```diff
@@ -277,6 +277,14 @@
                         {
                             name: "criteo"
                         },
+                          {
+                            name: "uid20",
+                            storage: {
+                              type: "cookie",
+                              name: "uid",
+                              expires: 28
+                            }
```

Comment:
you are hardcoding the storage of the id within the module, so there's no reason a publisher should be setting storage since it's not used.